### ISSUE TITLE
Harvest Golem RotaryCraft canola Major duplication issues

### ISSUE THREAD

**Issue by dnt462** [Azanor/thaumcraft#1590]
I set a golem controlled RotaryCraft canola farm latter I go and check and the farm is covered in 20K+ canola seeds. I did a little testing earlier and managed to record the problem.

https://www.youtube.com/watch?v=iCsVy8Kh_LM

When testing with normal wheat I still revived a many hundred percent increase in canola seeds, possibly because the golem is still holding a canola seed? Testing in a creative world with a fresh golem and Wheat return normal vanilla drop rates.

I'm thinking it relates to the canola seeds planting mechanic where it will use a seed when you right click, and if it can't plant will drop as an item.

Tested with TC 4.2.3.4, RotaryCraft v3d, forge 1291.

**Comment by shobu9** [comment]
#Blame Reika I really didn't have the issues maybe try upgrading forge  ? I have no idea how that happened...

**Comment by Azanor** [comment]
Yup, the issue is more than likely on the RotaryCraft side. I just fake a player click on a block, but something on the actual plants side isn't treating the click correctly for a fake player.
